packages/css-library/src/stylesheets/elements.scss

```diff
@@ -5,3 +5,158 @@
 // p {
 //    @include @usa-paragraph
 // }
+
+@use "../../dist/tokens/scss/variables" as *;
+
+$color-link-default-hover: rgba($color-black, 0.05);
+$small-screen:  481px;  // TODO: USWDSv3 breakpoints?
+$large-screen:  1201px; // TODO: USWDSv3 breakpoints?
+
+html,
+body {
+  padding: 0;
+  margin: 0;
+}
+
+html {
+  font-weight: 500;
+}
+
+body {
+  background: $color-white;
+  color: $color-base;
+}
+
+article > h1 {
+  margin-top: 0;
+}
+
+hr {
+  margin: 2.5em 0;
+  margin: 48px 0 40px; // converted to 16px base font: 3rem 0 2.5rem
```

Comment:
I'm converting `rem` to pixels so that it's compatible with both 10px and 16px base fonts.